app/components/appMainModule/localePageModule/electionTypePage/electionTypePageDirective.js

```diff
@@ -1,3 +1,5 @@
+'use strict';
```

Comment:
Most style guides put the "use strict" under the "(function() {", but maybe browserify encapsulates things so that doesn't matter? And it's probably fine to run everything in strict mode anyway.